geopandas_view/view.py

```diff
@@ -242,7 +242,16 @@ def view(
         )
 
     if column is not None:
-        if pd.api.types.is_categorical_dtype(gdf[column]):
+        if isinstance(column, (np.ndarray, pd.Series)):
+            if column.shape[0] != gdf.shape[0]:
+                raise ValueError(
+                    "The GeoDataframe and given column have different number of rows."
+                )
+            else:
+                column_name = '__plottable_column'
```

Comment:
This will show up in the tooltip/popup in some cases. We need to remove this string from `field` in `_tooltip_popup` as we do with `'__folium_key'`.

Reply:
Oops ! yes will do that